packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -357,15 +357,20 @@ function getFrameworkTests(opts: {
 			svelte: {
 				promptHandlers: [
 					{
-						matcher: /Which Svelte app template/,
+						matcher: /Which template would you like/,
 						input: [keys.enter],
 					},
 					{
-						matcher: /Add type checking with TypeScript/,
+						matcher: /Add type checking with Typescript/,
```

Comment:
It should be "TypeScript" not "Typescript", please revert.

Reply:
This is from sv not us